### ISSUE TITLE
No lr scheduler from configs necessary for training

### ISSUE THREAD

**Issue by bsmietanka** [runwayml/guided-inpainting#6]
First, thank you for sharing the code used for reproducing the results from your paper. I experimented a little bit and your solution seems promising.

One issue is that in your repository there is missing lr_scheduler.py file inside gi module. I took it from your other repository - taming-transformers and it seems to work (at least the training is in progress)

On the other hand, you mentioned in README that there is no propagation code provided. Could you maybe elaborate on how this would affect the performance of this solution on video sequences?

**Comment by bsmietanka** [comment]
Created a PR to solve issue with LR scheduler.

**Comment by dragen1860** [comment]
@bsmietanka hi, guy, have you succeed to run the code on custom video?
